packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -133,19 +133,21 @@ abstract class CachedArtifact {
   final Cache cache;
   final Platform platform;
 
+  /// The on-disk name of the artifact.
```

Comment:
Thanks for adding the inline comments!

Reply:
Honestly it's selfish. :) My future me wants them.